test/bin/build_images.sh

```diff
@@ -723,44 +679,11 @@ if [ -z "${release_info_rpm_base}" ]; then
     error "Failed to find microshift-release-info RPM in ${BASE_REPO}"
     exit 1
 fi
+
 SOURCE_VERSION=$(rpm -q --queryformat '%{version}' "${release_info_rpm}")
-MINOR_VERSION=$(echo "${SOURCE_VERSION}" | cut -f2 -d.)
-PREVIOUS_MINOR_VERSION=$(( "${MINOR_VERSION}" - 1 ))
-YMINUS2_MINOR_VERSION=$(( "${MINOR_VERSION}" - 2 ))
-FAKE_NEXT_MINOR_VERSION=$(( "${MINOR_VERSION}" + 1 ))
 SOURCE_VERSION_BASE=$(rpm -q --queryformat '%{version}' "${release_info_rpm_base}")
-
-current_version_repo=$(get_rel_version_repo "${MINOR_VERSION}")
-CURRENT_RELEASE_VERSION=$(echo "${current_version_repo}" | cut -d, -f1)
-CURRENT_RELEASE_REPO=$(echo "${current_version_repo}" | cut -d, -f2)
-
-previous_version_repo=$(get_rel_version_repo "${PREVIOUS_MINOR_VERSION}")
-PREVIOUS_RELEASE_VERSION=$(echo "${previous_version_repo}" | cut -d, -f1)
-PREVIOUS_RELEASE_REPO=$(echo "${previous_version_repo}" | cut -d, -f2)
-
-RHOCP_MINOR_Y=""
-RHOCP_MINOR_Y_BETA=""
-if is_rhocp_available "${MINOR_VERSION}"; then
-    RHOCP_MINOR_Y="${MINOR_VERSION}"
-elif is_rhocp_beta_available "${MINOR_VERSION}"; then
-    RHOCP_MINOR_Y_BETA="https://mirror.openshift.com/pub/openshift-v4/${UNAME_M}/dependencies/rpms/4.${MINOR_VERSION}-el9-beta/"
-fi
-
-RHOCP_MINOR_Y1=""
-RHOCP_MINOR_Y1_BETA=""
-if is_rhocp_available "${PREVIOUS_MINOR_VERSION}"; then
-    RHOCP_MINOR_Y1="${PREVIOUS_MINOR_VERSION}"
-elif is_rhocp_beta_available "${PREVIOUS_MINOR_VERSION}"; then
-    RHOCP_MINOR_Y1_BETA="https://mirror.openshift.com/pub/openshift-v4/${UNAME_M}/dependencies/rpms/4.${PREVIOUS_MINOR_VERSION}-el9-beta/"
-fi
-
-# For Y-2, there will always be a real repository, so we can always
-# set the template variable for enabling that package source and use
-# the well-known name of that repo instead of figuring out the URL.
-yminus2_version_repo=$(get_rel_version_repo "${YMINUS2_MINOR_VERSION}")
-YMINUS2_RELEASE_VERSION=$(echo "${yminus2_version_repo}" | cut -d, -f1)
-YMINUS2_RELEASE_REPO="$(get_ocp_repo_name_for_version ${YMINUS2_MINOR_VERSION})"
-RHOCP_MINOR_Y2="${YMINUS2_MINOR_VERSION}"
+export SOURCE_VERSION
+export SOURCE_VERSION_BASE
```

Comment:
Why are those variables not in common_version.sh? They're duplicated in build_bootc_images.py

Reply:
To calculate these vars, we need access to LOCAL_REPO, so there will be dependency on common.sh file, which I wanted to avoid.

Reply:
I can add it if you think it's a good idea.

Reply:
I don't know. We removed duplication in other places so it's a bit asymmetrical, but I'll leave decision to you, you have better knowledge of the refactor

Reply:
I think we should leave it the way it is. It's a minor code duplication, so I prefer not to introduce the dependency.
Maybe, we will do this later on.